typescript/src/optimistic-minting.ts

```diff
@@ -39,6 +39,22 @@ export type OptimisticMintingRequestedEvent = {
   fundingOutputIndex: number
 } & Event
 
+/**
+ * Represents an event that is emitted when an optimistic minting request
+ * is cancelled on chain.
+ */
+export type OptimisticMintingCancelledEvent = {
+  /**
+   * Guardian's chain identifier.
+   */
+  guardian: Identifier
+  /**
+   * Unique deposit identifier.
+   * @see Bridge.buildDepositKey
+   */
+  depositKey: BigNumber
```

Comment:
Could this be a `Hex`? `buildDepositKey` returns a string with `0x` prefix.

Reply:
A deposit key is an `uint256` in the `Bridge` contract so we should rather make `buildDepositKey` return a `BigNumber`. Left a TODO about that: d6e9352b

Reply:
The value of the deposit key is a result of `solidityKeccak256`. It feels natural to store the digest as hex.

Reply:
Honestly, I don't see any serious difference in how we handle it within typescript, we can always display both representations according to the given case. I just wanted to keep it aligned with the contract-side representation but if you think it will be better to keep it as `Hex`, here it is: 30587e04

Reply:
Thanks!